cf_units/tests/test_unit.py

```diff
@@ -375,6 +375,10 @@ def test_multiply_with_unknown_unit(self):
         v = Unit('meters')
         self.assertTrue((u * v).is_unknown())
         self.assertTrue((v * u).is_unknown())
+        self.assertEqual((u * v).origin, unit._UNKNOWN_UNIT_STRING)
```

Comment:
This is an implementation detail that I'd like to keep out of the "multiply" test, if possible.

One thing I'd be tempted to do is to define a ``assertIsUnknown`` method, which can do these checks that you are running but without the drag of needing to define what an unknown unit should actually represent in multiple places.